cdap-ui/app/cdap/components/Cloud/Profiles/ListView/index.js

```diff
@@ -110,19 +110,37 @@ class ProfilesListView extends Component {
   static propTypes = {
     namespace: PropTypes.string.isRequired,
     profiles: PropTypes.array,
+    defaultProfile: PropTypes.string,
```

Comment:
Are these two props being used anywhere we use `<ProfilesListView />`?

Reply:
Yeah, we need the `defaultProfile` prop to know which profile is the default for that level